cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -456,6 +657,91 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
         );
     }
 
+    private renderTrackingContent(): JSX.Element {
+        const { activeLabelID, trackers, activeTracker } = this.state;
+        const {
+            labels, canvasInstance, onInteractionStart, frame, jobInstance,
+        } = this.props;
+        if (!trackers.length) {
+            return (
+                <Row justify='center' align='middle' style={{ marginTop: '5px' }}>
```

Comment:
Please, put static styles to scss files. Also round marginTop till $grid-unit-size (8px).

Reply:
Done